tests/com/tony/server/response/ImageContentResponseTest.java

```diff
@@ -42,8 +42,8 @@ public void testImageTypeReturnsCorrectTypeGivenPath() throws Exception {
     @Test
     public void testRespondsWith500ErrorIfNoSuchFile() throws Exception {
         ImageContentResponse imageContentResponse = new ImageContentResponse(System.getProperty("user.dir") + "/noimage.png");
-        String expectedResponse = "HTTP/1.1 500 Internal Server Error\n" +
-                "\n" +
+        String expectedResponse = "HTTP/1.1 500 Internal Server Error\r\n" +
+                "\r\n" +
                 "java.nio.file.NoSuchFileException: /Users/tony/Developer/java-server/noimage.png";
```

Comment:
This seems like a pretty fragile test, since my I don't have a `/Users/tony/Developer/java-server/` directory on my computer.